resources/newssite/news-nuxt/composables/provide-locale.js

```diff
@@ -16,10 +18,41 @@ export function provideLocale() {
         htmlAttrs: { dir, lang },
     });
 
+    const { content } = dataSource[lang];
+
+    const selected = Object.create(null);
```

Comment:
I'm not sure where this term of "selected" comes from, where we apply this to all contents. I'd use `contentWithIds` or something similar for more clarity.